Pomodoro/app/src/main/java/no/uia/ikt205/pomodoro/MainActivity.kt

```diff
@@ -15,8 +15,9 @@ class MainActivity : AppCompatActivity() {
     lateinit var startButton:Button
     lateinit var coutdownDisplay:TextView
 
-    val timeToCountDownInMs = 5000L
+    var timeToCountDownInMs = 5000L
     val timeTicks = 1000L
+    val oneMinutinTicks = 60000L;
```

Comment:
En idé her for å gjøre det enda mer forståelig hva 60000L er kunne du ha laget en funksjon som konverterer millisekunder til minutter. Ellers gode variabel navn som er lett å forstå 😊